src/lib/npm-ci.js

```diff
@@ -4,7 +4,12 @@ import { rmRF } from '@actions/io';
 import * as log from './log.js';
 import exec from './exec.js';
 
-async function npmCi({ cwd } = {}) {
+async function npmCi({ cwd, installCommand } = {}) {
+	if (!installCommand) {
+		installCommand = autoDetectInstallCommand();
```

Comment:
Let's default this to `npx ci`: https://github.com/privatenumber/ci

Reply:
That package is not easily importable, would it be possible to export the function in a way that can be imported by other libs instead of just it being a self executed script?